FAQ.md

```diff
@@ -0,0 +1,56 @@
+---
+status: draft
+---
+
+# Frequently Asked Questions (FAQs)
+
+## Why create a new foundation?
+
+There are already several foundations which host open source projects,
+and each has its own focus and requirements.
+The CF is a home for self-governing, code-centric open source projects.
+Each open source project has its own unique needs and goals,
+and not every foundation is the right fit for every project.
+
+There are many critical application frameworks and libraries within the open source ecosystem that are not currently part of a foundation.
+The CF is a new option for projects to consider when looking for a foundation to call home.
+Helping to strengthen the open source ecosystem as a whole by ensuring more projects are sustainable in the long term.
```

Comment:
This section and the ones below mostly give reasons not to join other foundations, but no reason for joining CH... it's a bit weird, no?
As things are, it looks like not joining any foundation would be just as good a solution as joining CH.

Maybe we need at least a few arguments in favor of joining CH? I.e. "you'll get this", "you can hope for that", ...

Reply:
I guess I agree with @yrodiere. And I'm not really convinced we even need to have a list of the "competition". Why not just remove all that and have a positive list of "differentiators", which doesn't need to be explicit about _who_ each point differentiates us from.

Reply:
Concerns were raised that not having an FAQ would have people wondering why not one of the other foundations.

If there's a consensus we don't need a deliberate FAQ answering these questions,  https://github.com/commonhaus/foundation-draft/blob/main/README.md covers the positives of the foundation.

Reply:
That readme seems a bit vague and high-level to me, as I'd have expected something like "we provide infra" or "we provide access to lawyers" or "we provide spronsorship processes" or...

But anyway. I think having sections that explain "why not Apache/Eclipse/etc." makes sense, it's just that "Why create a new foundation?" would need some reason for the foundation to exist. Linking to the readme might provide that information?

Reply:
I've updated the "Why create a new foundation?" with links to the specific details in the README

Reply:
Thanks @kenfinnigan , that looks good.

@ebullient Regarding the "why CH"... While I can understand the reluctance to over-promise, I think we agree there is a need to promise *something*.

Dreaming requires a fair amount of optimism, which I'm afraid is generally not my _forte_.

I can summarize what I got from previous readings and discussions though, by order of importance:

1. Independent, vendor-neutral IP ownership, with clear processes for both coordinating upstream projects and giving vendors controlled but fair influence. This is with the goal of fostering cooperation between multiple interested parties, as opposed to projects that seem "owned" by a company and may "repel" other companies.
2. Foundational rules: code of conduct, ...
3. Structure for small/starting projects that need it (a GitHub org, basically?)

Reply:
Right - no need to make specific promises, but it would be nice to picture a clarer vision of what you'd hope this to become.
Some principles are nicely expressed already and I got a sense of why you're working on this, but I think it would help to make it a bit more concrete; perhaps some examples of what you'd hope to see coming? It's all sounding nice but it's also still quite vague.